sdk/eventgrid/eventgrid/src/eventGridClient.ts

```diff
@@ -9,20 +9,22 @@ import {
   OperationOptions
 } from "@azure/core-http";
 
-import { createHmac } from "crypto";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
instead of depending on `uuid` here you could use `generateUuid()` from `@azure/core-http` (that already depends on uuid)